kl_conferences/serializers/server_node_serializer.py

```diff
@@ -11,14 +14,24 @@ class Meta:
         fields = '__all__'
 
 
+class HostnameField(serializers.CharField):
+    max_length = 128
+
+    def to_internal_value(self, data):
+        data = str(data)
+        if not re.match(r'^([\w\d]+\.)+\w+(:\d+)?$', data):
+            raise ValidationError('Incorrect hostname format!')
+        return data
+
+
 class ServerNodeRegisterRequestSerializer(serializers.Serializer):
-    hostname = serializers.URLField()  # "http://bbb1.agooddomain.pl/",
+    hostname = HostnameField() # "http://bbb1.agooddomain.pl/",
```

Comment:
comment is wrong, HostnameField doesn't allow full uri

Reply:
It's fine - comment leads to confusion - it will be hostname